projects/homework_102_nourish/src/main.c

```diff
@@ -0,0 +1,44 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include "interrupt.h"   // interrupts are required for soft timers
+#include "log.h"         // for printing
+#include "soft_timer.h"  // for soft timers
+#include "wait.h"        // for wait function
+
+#define INTERVAL_COUNTER_A 500
+#define INTERVAL_COUNTER_B 1000
+
+typedef struct {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Main_var;
```

Comment:
The FW 102 homework specification does specify that the struct should be named `Counters`. Also, generally we both name the struct and typedef it, so this should be:
```c
typedef struct Counters {
  uint8_t counter_a;
  uint8_t counter_b;
} Counters;
```
Finally, type names should be in `UpperCamelCase`.